payments/mobile-pay/index.md

```diff
@@ -16,214 +16,63 @@ sidebar:
-* Directions to (a link to a page) the merchant's terms and conditions
-  (such as return policy) and information of how the consumer can contact the
-  merchant.
-* Details informing the consumer that he or she accepts the Terms & Conditions
-  when clicking on the Payment Link.
-
-## Recommendations about receipt
-
-We recommend that you send an e-mail or SMS confirmation with a receipt to
-the consumer when the payment has been fulfilled.
-
-## API requests
-
-The API requests depend on the payment instrument you are using when implementing
-the Payment Link scenario, see [purchase flow](#purchase-flow).
-One-phase payment metods will not implement capture, cancellation or reversal.
-The options you can choose from when creating a payment with key operation set
-to Value Purchase are listed below.
-
-## Screenshots
-
-When clicking the payment link, the consumer will be directed to a payment
-page, similar to the examples below, where payment information can be entered.
-
-![Payex payment menu][paymentMethodsScreenshot]
-
-### Options
-
-All valid options when posting in a payment with operation Purchase,
-are described in each payment instrument's respective API reference.
-Please see the general sequence diagrams for more information about payments
-in one-phase (e.g. [Swish][swish-m-commerce] and credit card with autocapture)
-and two-phase (e.g. [Credit card][credit-card],
-[MobilePay][mobile-pay], [Vipps][vipps]).
-
-#### Authorization
-
-* PreAuthorization (Credit card):
-  * If you specify that the _intent_ of the _purchase_ is PreAuthorize,
-   it's almost the same as an authorization,
-    **except that no money will be reserved** from the consumers credit card,
-   [before you make a finalize on this transaction (using the PATCH on
-   the Autorization)][tecnical-reference-finalize].
-
-* Authorize (two-phase):
-  * When using two-phase flows you reserve the amount with an authorization,
-   you will have to specify that the intent of the purchaseis Authorize.
-   The amount will be reserved but not charged.
-   You will later (i.e. when you are ready to ship the purchased products)
-   have to make a `Capture` or `Cancel` request.
-
-#### Capture
-
-*Autocapture (one-phase credit card)*:
-If you want the credit card to be charged right away, you will have to specify
-that the intent of the purchaseis Autocapture.
-The credit card will be charged and you don't need to do any more financial
-operations to this purchase.
-
-#### General
-
-When implementing the Payment Link scenario, it is optional to set a
-[CallbackURL][technical-reference-callback] in the `POST` request.
-If callbackURL is set Swedbank Pay will send a postback request to this URL
-when the consumer as fulfilled the payment.
+{% include jumbotron.html body="  Enter your phone number,
+get the push message, press the pay button... and you're done!  " %}
+
+Mobile Pay is the main Danish payment app for mobile phones, making it one of
```

Comment:
Is it `MobilePay` or `Mobile Pay`? From [mobilepay.dk](https://mobilepay.dk/) it looks like the former is correct.

```suggestion
MobilePay is the main Danish payment app for mobile phones, making it one of
```